membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -76,10 +76,19 @@ class AttributeController(
       latestOneOffDate: Option[LocalDate],
       mobileSubscriptionStatus: Option[MobileSubscriptionStatus],
   ): Attributes = {
-    val mobileExpiryDate = mobileSubscriptionStatus.map(_.to.toLocalDate)
+    // Filter out Feast subscriptions from Live App subscription expiry date
+    val liveAppMobileExpiryDate = mobileSubscriptionStatus.flatMap { status =>
+      val isFeastSubscription = status.productId.exists(_.contains("Feast"))
```

Comment:
I've found a live productId with a lowercase 'f' so I would suggest making the check case insensitive.